iosxr_iso2vbox.py

```diff
@@ -178,15 +178,17 @@ def xr_cli_wait_for_output(command, pattern):
 
         for attempt in range(total):
             try:
+                logger.debug("Looking for '%s' in output of '%s'\n" % (pattern, command))
```

Comment:
Having `\n` at the end of a log message just results in an extra blank line after this message, as log messages are already newline-terminated - are you really sure you want this?